src/components/MediaPlayerCarousel.tsx

```diff
@@ -109,28 +111,25 @@ export class MediaPlayerCarousel extends React.PureComponent<Props, State> {
 
   _attemptBoost = () => {
     ReactNativeHapticFeedback.trigger('impactHeavy', HapticOptions)
-    this.explosion && this.explosion.start()
-    this.setState({ boostPaymentLoading: true }, () => {
-      (async () => {
-        try {
-          const { nowPlayingItem } = this.global.player
-
-          const { errors, transactions, totalAmountPaid } = await sendBoost(nowPlayingItem)
-
-          this.setState({ boostPaymentLoading: false })
-          this.setGlobal({
-            bannerInfo: {
-              show: true,
-              description: translate('Boost Sent'),
-              errors,
-              transactions,
-              totalAmount: totalAmountPaid
-            }
+    this.setState({ boostIsSending: true }, () => {
+      this.explosion && this.explosion.start()
+      try {
+        const { nowPlayingItem } = this.global.player
+        sendBoost(nowPlayingItem)
+        setTimeout(() => {
+          this.setState({
+            boostIsSending: false,
+            boostWasSent: true
+          }, () => {
+            setTimeout(() => {
+              this.setState({ boostWasSent: false })
+            }, 4000)
```

Comment:
Isn't this number a little arbitrary? What if you await for the sendBoost but don't do anything with the results, pass or fail...?

Reply:
It is arbitrary...since there are a bunch of reasons a transaction might fail, I'm thinking of just rendering "boost sent" without waiting for a network response. I can't think of a good reason to wait for one given current limitations?